styles.css

```diff
@@ -0,0 +1,763 @@
+body,
```

Comment:
- [ ] Be careful with the amount of important that you are using on this file. I would recommend you to create your own classes instead of modifying the ones that boostrap has. Kindly work on that as suggested by the previous code reviewer.